README.md

```diff
@@ -1,5 +1,11 @@
 # Vault Helm Chart
 
+---
+**Please note**: We take Vault's security and our users' trust very seriously. If 
+you believe you have found a security issue in Vault Helm, _please responsibly disclose_ 
+by contacting us at [security@hashicorp.com](mailto:security@hashicorp.com).
+---
```

Comment:
I think there should be a space between line 6 and 7, otherwise the entire paragraph is big and bold like a header:

![image](https://user-images.githubusercontent.com/444265/83790602-4e071100-a64d-11ea-9c2c-784b839939df.png)

I'd also suggest something like this:
```suggestion
> :warning: **Please note**: We take Vault's security and our users' trust very seriously. If 
you believe you have found a security issue in Vault Helm, _please responsibly disclose_ 
by contacting us at [security@hashicorp.com](mailto:security@hashicorp.com).
```

![image](https://user-images.githubusercontent.com/444265/83790498-26b04400-a64d-11ea-9292-9241da0d6af0.png)